src/main/java/ch/wisv/areafiftylan/dto/SeatGroupDTO.java

```diff
@@ -0,0 +1,25 @@
+package ch.wisv.areafiftylan.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Created by sille on 10-1-16.
+ */
+public class SeatGroupDTO {
```

Comment:
`@Getter` and `@Setter` can be omitted if you add `@Data` to the class, which also gives to `#toString()`, `#equals()` and `#hashcode` for free (which is nice in collections and test matchers).